### PR TITLE
[CT-629] Fix entryPrice calc

### PR DESCRIPTION
**Pull request by dydxwill**
## Summary by CodeRabbit

- **Bug Fixes**
	- Adjusted test cases for `LiquidationHandler` and `OrderHandler` to reflect changes in the `defaultPerpetualPosition` size, ensuring accurate validation of liquidation orders and order fills.
	- Updated calculations in tests to align with the new expected values for `sumOpen` and position size.

- **Chores**
	- Refined SQL functions for handling entry price calculations, improving accuracy in order fill processing and perpetual position updates.
	- Enhanced test suite by adding new scenarios for order fills, ensuring thorough validation of order handling logic.

### CODE REVIEW

File: indexer/services/ender/src/scripts/helpers/dydx_liquidation_fill_handler_per_order.sql [hunk 1]
```diff
@@ -200,7 +200,7 @@ BEGIN
                 perpetual_position_record."side", order_side) THEN
             sum_open = dydx_trim_scale(perpetual_position_record."sumOpen" + fill_amount);
             entry_price = dydx_get_weighted_average(
-                    perpetual_position_record."entryPrice", perpetual_position_record."sumOpen",
+                    perpetual_position_record."entryPrice", ABS(perpetual_position_record."size"),
```
Comment (vincentwschau) [inline indexer/services/ender/src/scripts/helpers/dydx_liquidation_fill_handler_per_order.sql:203]: nit: Possible to add a test-case where perpetual position size is = negative size of fill to test that the previous error case couldn't happen again?

**Review by vincentwschau — APPROVED** [review]

**Comment by dydxwill** [comment]
@mergifyio backport release/indexer/v6.x
